apps/test/storybook/util/testStorybook.js

```diff
@@ -66,6 +69,18 @@ class FakeStorybook {
     return this;
   }
 
+  withReduxStore(reducers = {}) {
```

Comment:
Can we deduplicate this implementation with the real storybook one?  They seem to be the same, and I'd hate for these to diverge later.